enter/dglenter/pipeline/graphpred/gen.py

```diff
@@ -0,0 +1,105 @@
+from pathlib import Path
+from jinja2 import Template
+import copy
+import typer
+from pydantic import BaseModel, Field
+from typing import Optional
+import yaml
+from ...utils.factory import PipelineFactory, NodeModelFactory, PipelineBase, DataFactory
+from ...utils.base_model import EarlyStopConfig, DeviceEnum
+from ...utils.yaml_dump import deep_convert_dict
+import ruamel.yaml
+from ruamel.yaml.comments import CommentedMap
+
+class NodepredPipelineCfg(BaseModel):
+    node_embed_size: Optional[int] = -1
+    early_stop: Optional[EarlyStopConfig] = EarlyStopConfig()
+    num_epochs: int = 200
+    eval_period: int = 5
+    optimizer: dict = {"name": "Adam", "lr": 0.005}
+    loss: str = "CrossEntropyLoss"
+@PipelineFactory.register("nodepred")
+class NodepredPipeline(PipelineBase):
```

Comment:
Consider delete the entire graphpred folder as we are cutting it off. Remember to save it for later develop.